src/api/index.ts

```diff
@@ -1,11 +1,17 @@
 import express from 'express'
 import adminRouter from './admin'
 import mappingRouter from './mapping'
+import { getAvailableDomains } from './lib/data'
```

Comment:
I think this is from the last PR.

Reply:
@allopez7  Correct I should precise that PR rely on #143